examples/plugin/src/noarg/BUILD

```diff
@@ -2,10 +2,12 @@ load("//kotlin:kotlin.bzl", "kt_compiler_plugin", "kt_jvm_library", "kt_jvm_test
 
 kt_compiler_plugin(
     name = "no_arg_plugin",
+    compile_phase = True,
```

Comment:
Setting these to default=true makes even more sense when you see all of these additions. It is less explicit than ex-googlers want to be, but most people in the wild are going to be naive users of published plugins, not necessarily build experts.